examples/static_files/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use std::fs::File;
+use std::io::Read;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::response::Body;
+
+use super::rocket;
+
+fn body_to_bytes(body: Body<&mut Read>) -> Option<Vec<u8>> {
+    match body {
+        Body::Sized(b, _)  => {
+            let mut data: Vec<u8> = vec![];
+            b.read(&mut data).expect("Read from response failed");
+            return Some(data);
+        },
+        Body::Chunked(b, _) => {
+            let mut data: Vec<u8> = vec![];
+            b.read(&mut data).expect("Read from response failed");
+            return Some(data);
+        }
+    }
+}
+
+fn test_query_file<T> (path: &str, status: Status, content: T)
+    where T: Into<Option<Vec<u8>>>
+{
+    let rocket = rocket();
+    let mut req = MockRequest::new(Get, &path);
+
+    let mut response = req.dispatch_with(&rocket);
+    assert_eq!(response.status(), status);
+
+    let body_data = response.body().and_then(|body| body_to_bytes(body));
+    if let Some(expected_data) = content.into() {
+        assert!(body_data.map_or(false, |s| s == expected_data));
+    }
+}
+
+fn read_file_content(path: &str) -> Vec<u8> {
+    let mut fp = File::open(&path).expect(&format!("Can not open {}", path));
+
```

Comment:
Kill this empty line. Move the `file_content` declaration to the top of the function.